src/main/webapp/script.js

```diff
@@ -29,30 +29,60 @@ function loadTasks() {
 }
 
 function renderTasks(tasks) {
-  // TODO: check the size of tasks and show a warning if zero
-  $("#task-list").empty();
-  tasks.forEach(renderSingleTask);
+  const taskList = $("#task-list");
+  taskList.empty();
+  if (tasks.length > 0) {
+    $("#schedule-button").prop("disabled", false);
```

Comment:
You can extract this one before the if and do `.prop("disabled", !tasks.length);`